src/app/settings/user-subscription.component.ts

```diff
@@ -210,4 +210,8 @@ export class UserSubscriptionComponent implements OnInit {
   get usingInAppPurchase() {
     return this.sub != null ? this.sub.usingInAppPurchase : false;
   }
+
+  get title(): string {
+    return this.i18nService.t(this.selfHosted ? "subscription" : "premiumMembership");
```

Comment:
Is this in the wireframes? I don't understand why we'd change the title this way.